README.md

```diff
@@ -129,6 +129,7 @@ Recipe | Description
 [Code coverage](https://github.com/paulfalgout/cypress-coverage-example) | Cypress with Coverage reports
 [Cucumber](https://github.com/TheBrainFamily/cypress-cucumber-example) | Example usage of Cypress with Cucumber
 [Jest](https://github.com/TheBrainFamily/jest-runner-cypress-example) | Example for the jest-runner-cypress
+[Mailosaur](https://github.com/muratkeremozcan/cypressExamples/tree/master/cypress-mailosaur) | Utilizes `cy.request` or `cy.task` with [Mailosaur's Node package](https://www.npmjs.com/package/mailosaur) to test emails sent with [sendmail npm package](https://www.npmjs.com/package/sendmail)
```

Comment:
Commands referenced with command invocation - `cy.request` to `cy.request()` - shorten references to packages. 

```suggestion
[Mailosaur](https://github.com/muratkeremozcan/cypressExamples/tree/master/cypress-mailosaur) | Utilizes `cy.request()` or `cy.task()` with [`mailosaur`](https://www.npmjs.com/package/mailosaur) to test emails sent with [`sendmail`](https://www.npmjs.com/package/sendmail)
```

Reply:
adjusted cy commands referenced with command invocation, shortened package names in [commit](https://github.com/cypress-io/cypress-example-recipes/pull/469/commits/7b12caff18003f1c67567afbea869d26fbf34860).